evm/src/coverage/visitor.rs

```diff
@@ -0,0 +1,397 @@
+use super::{BranchKind, CoverageItem, SourceLocation};
+use ethers::{
+    prelude::sourcemap::SourceMap,
+    solc::artifacts::ast::{self, Ast, Node, NodeType},
+};
+use std::collections::HashMap;
+use tracing::warn;
+
+#[derive(Debug, Default, Clone)]
+pub struct Visitor {
```

Comment:
this is a Visitor for the `AST` object that solc includes in the output, right?

Reply:
yes